src/platform/testing/e2e/nightwatch-commands/log.js

```diff
@@ -3,7 +3,7 @@
  *  logging as the browser is running through the test due to the way nightwatch
  *  runs all the code beforehand.
  *
- * @param {String} message The message you want to log to the console
+ * @param {String} [message=''] The message you want to log to the console
```

Comment:
Capitalized `String` or lowercase `string`? Looks like you were going for the latter for consistency.